client/src/utils/application/nLinearStepperValidation.js

```diff
@@ -9,6 +9,20 @@ const generalInfosSchema = yup.object().shape({
     .required('Mobile Number is required'),
   age: yup.string().required('Age is required').trim(),
   address: yup.string().required('Address is required'),
+  motivation: yup.string().required(`Insert motivation`).trim(),
+  specialization: yup.string().required(`specialization`).trim(),
+  university: yup.string().required(`university`).trim(),
+  codingExperience: yup.string().required(`codingExperience`).trim(),
+  primaryMotivation: yup.string().required(`primaryMotivation`).trim(),
+  caReading: yup.string().required(`caReading`).trim(),
```

Comment:
its from where the applicant hear about code academy in GSG
```suggestion
  caReaching: yup.string().required(`caReaching`).trim(),
```